extensions/wikia/Email/Controller/WelcomeController.class.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Check;
+use Email\Tracking\TrackingCategories;
+use Email\ImageHelper;
+
+class WelcomeController extends EmailController {
+	const TRACKING_CATEGORY = TrackingCategories::WELCOME;
+	const MOBILE_URL_IOS = 'https://itunes.apple.com/us/artist/wikia-inc./id422467077';
+	const MOBILE_URL_ANDROID = 'https://play.google.com/store/apps/developer?id=Wikia,+Inc';
+
+	protected function getSubject() {
+		return $this->getMessage( 'emailext-welcome-subject', $this->getCurrentUserName() )->parse();
+	}
+
+	/**
+	 * @template digestLayout
+	 */
+	public function body() {
+		$this->response->setData( [
+			'salutation' => $this->getSalutation(),
+			'summary' => $this->getSummary(),
+			'details' => $this->getDetailsList(),
+			'contentFooterMessages' => [
+				$this->getMessage( 'emailext-welcome-footer-community' )->parse(),
+				$this->getMessage( 'emailext-welcome-footer-thanks' )->text()
+			],
+			'hasContentFooterMessages' => true,
+		] );
+	}
+
+	protected function getSummary() {
+		return $this->getMessage( 'emailext-welcome-summary' )->text();
+	}
+
+	protected function getDetailsList() {
+		if ( in_array( $this->targetLang, [ 'de', 'en', 'es', 'ja', 'pt', 'ru', 'zh' ] ) ) {
```

Comment:
For Chinese we support the ```targetLang``` will be 'zh-hans' or 'zh-tw'.  However, there's still the issue that the URLs generated below won't be localized to the language, e.g. ```http://community.wikia.com/wiki/Special:Videos``` for English and ```http://comunidad.wikia.com/wiki/Especial:V%C3%ADdeos``` for Spanish.  I would recommend moving this URL to the i18n file and leaving instructions in the LOC ticket about translating the URL per language.  At-mark Cristina and Bert to help define those.